CONTRIBUTING.md

```diff
@@ -95,6 +95,8 @@ For any change that is customer-facing and should appear in the changelog file,
 
 When composing commit messages, it's important to use past tense. Additionally, if there is a corresponding ticket or issue associated with the change, please put the ticket link in the commit message and pull request.
 
+Keep commit messages within 100 characters to comply with the current CI pipeline restriction.
```

Comment:
This is not restricted by CI pipeline.
See https://github.com/instana/nodejs/blob/v3.14.4/commitlint.config.js

Reply:
Thank you for highlighting the configuration. 
We have checking this restriction at the CI pipeline level, rather than limiting it at the commit time, this what I meant. Will update.